src/components/pool_flux_chart.vue

```diff
@@ -32,14 +33,17 @@
           <label><input type="radio" name="x-scale" value="log" checked> log </label>
           <label><input type="radio" name="x-scale" value="linear"> linear </label>          
         </form>
-        <p>You are probably familiar with linear scales. When you move a fixed distance on a linear axis, you add a fixed value to your starting value. When you move a fixed distance on a log axis, you multiply the starting value by a value of 10.</p>
+        <p :text="axisExplanation">{{ axisExplanation }}</p>
+        <p><span class='poolText' >Pools</span> are places where water is stored, like the ocean. <span class='fluxText'>Fluxes</span> are the ways that water moves between pools, such as evaporation, precipitation, discharge, recharge, or human use.</p>
         <p>The data for this chart are adapted from <a href="https://www.nature.com/articles/s41561-019-0374-y" target="_blank">Abbott et al. (2019) Human domination of the global water cycle absent from depictions and perceptions.</a></p>
+        <p>Learn more about the water cycle and see the water cycle diagram on the <a href="https://www.usgs.gov/water-cycle" target="_blank">USGS Water Science School website.</a></p>
```

Comment:
The link to the WSS could also be at the top and more prominent, leaving just the scale thing below as a fig caption. That way more people will be likely to click through